src/app/pages/send/send-crypto-asset-form/hooks/use-font-resizer.ts

```diff
@@ -10,6 +10,8 @@ export const amountInputId = 'amount-input';
 export const assetSymbolId = 'asset-symbol';
 export const maxInputContainerWidth = 344;
 
+// TODO: This really isn't working consistently
+// Will need to revisit handling different symbols
```

Comment:
why is the currency symbol async in the first place? 🤔

Reply:
We have to wait for the DOM element to load so we can get the text width (`offsetWidth`), but it doesn't seem to work  consistently. This is all made way more difficult bc we have to resize/center the symbol with the input as the user types ...and the number of characters in the symbol is unknown.